riotgen/tests/test_data/driver/driver.c

```diff
@@ -19,8 +19,8 @@
  */
 
 #include "test.h"
-#include "test_constants.h"
-#include "test_params.h"
+#include "include/test_constants.h"
```

Comment:
This is not the way things are usually done in RIOT. The riotgen tool is not made to fully automatize driver code creation just help with the boilerplate things that are part of the driver itself.
The proper fix is not to patch riotgen but rather to find a way in RIOT to add this include directory from a Makefile.include added in the driver itself and parsed automatically by the RIOT buildsystem when the module is used.